packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -747,7 +747,7 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
         // System messages may have no clientId (but some do, like propose, noop, summarize)
         assert(
             message.clientId !== undefined
-            || isSystemMessage(message),
+            || !isClientMessage(message),
```

Comment:
This is correct change, but please it's a change in behavior. So we should validate nothing breaks.
Also comment above becomes wrong with this change.
And I'd test if we could change this assert to be (message.clientId !== undefined) === isClientMessage(message)
That's more correct and stricter assert, but maybe there is an exception to it - I'd test it on examples to learn and see if we can make such change.